### PR TITLE
Administration page

### PR DESCRIPTION
**Pull request by marekm294**

### CODE REVIEW

File: lib/Models/Information.dart [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  Map<String, String> toUpdateMap() {
+    return {idColumn : id.toString(), titleColumn: title, descriptionColumn: description};
+  }
+
+  static Information fromInsertMap(Map<String, String> map) {
+    return Information(title: map[titleColumn]!, description: map[descriptionColumn]!);
+  }
+
+  static Information fromMap(Map<String, String> map) {
```
Comment (miakh) [inline lib/Models/Information.dart:38]: remove

File: lib/pages/AdministrationPage.dart [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+          columns: columns,
+          rows: rows,
+          onChanged: (PlutoGridOnChangedEvent event) {
+            stateManager.notifyListeners();
+          },
+          onLoaded: (PlutoGridOnLoadedEvent event) {
+            stateManager = event.stateManager;
+          },
+          rowColorCallback: (rowContext){
+            // return rowContext.row.state == PlutoRowState.added ? Colors.orange : Colors.transparent;
```
Comment (miakh) [inline lib/pages/AdministrationPage.dart:71]: remove

File: lib/pages/AdministrationPage.dart [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+            ]
+        ),
+      ),
+    );
+  }
+
+  void _reloadDataAsync() async{
+    var information = await supabase
+      .from("information")
+      .select() ;
```
Comment (miakh) [inline lib/pages/AdministrationPage.dart:130]: orderby desc created_at

Comment (marekm294) [reply lib/pages/AdministrationPage.dart]: ordered

**Pull request merged**